trending_data/src/main/java/com/google/sps/servlets/Search.java

```diff
@@ -70,6 +70,11 @@ public void initialize(HttpRequest request) throws IOException {}
       List<Video> searchResultList = searchResponse.getItems();
 
       if (searchResultList != null) {
+        if (searchResultList.size() == 0) {
```

Comment:
isEmpty() makes it a bit easier to read. But it looks like this if isn't necessary at all. If it is empty, it returns an empty list one line later anyone.

Reply:
Oh that's true, the function `convertToYTVid` will return an empty list if there's no videos in the response.